addons/bestja_offers/templates.xml

```diff
@@ -51,40 +51,49 @@
 
                     <section class="row">
                         <section class="offer_content col-md-5 col-md-offset-1">
-                            <h2>Dlaczego potrzebujemy pomocy</h2>
-                            <section t-if="offer.description">
-                                <p>Celem projektu jest: <span t-field="offer.description"></span></p>
+                            <section t-if="offer.desc_aim">
```

Comment:
You change the current names of those fields to the OLD names of those fields. This won't work. Pull our most recent changes and start using the current field names.